proxysql-admin

```diff
@@ -76,8 +76,10 @@ declare    SLAVE_IS_WRITER=""
 declare    MODE="singlewrite"
 declare -i NODE_CHECK_INTERVAL=3000
 
-declare -i WRITE_HOSTGROUP_ID=10
-declare -i READ_HOSTGROUP_ID=11
+declare -i READ_HOSTGROUP_ID=10
+declare -i WRITE_HOSTGROUP_ID=11
+declare -i BACKUP_WRITE_HOSTGROUP_ID=12
+declare -i OFFLINE_HOSTGROUP_ID=13
```

Comment:
@delgod We should add `BACKUP_WRITE_HOSTGROUP_ID` and `OFFLINE_HOSTGROUP_ID` in proxysql-admin.cnf. If we need to add two PXC cluster in proxysql db there will be hostgroup conflict. We can give different hostgroups if we have these variables in proxysql-admin.cnf